blask-6/app/routes/blog.py

```diff
@@ -0,0 +1,33 @@
+from flask import Blueprint, render_template, redirect, url_for, request
+#last 3 imports are for the new form
```

Comment:
Ah, this was from when I was following along, and making note that `redirect`, `url_for`, and `request` were modules we imported to allow for the "adding post" functionality. Since it's not relevant for this unit, I'll clear it out.